### PR TITLE
Update read me file

### PR DESCRIPTION
**Pull request by chrismclellan1986**
- [Description](#description)
  - [Type of change](#type-of-change)
- [Checklist](#checklist)

# Description
Update to Read Me file with requirements on how to run the project, Set up and additional information

Fixes # (issue number)

## Type of change

Please delete options that are not relevant
- [x] This change requires a documentation update

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (if required)
- [x] My changes generate no new warnings
- [N/A] I have added tests that prove my fix is effective or that my feature works
- [N/A] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by Boopster — APPROVED** [review]
Suggest adding instructions for running node.js project - can we done under a separate PR

File: README.md [hunk 1]
```diff
@@ -1,5 +1,40 @@
...
+2. Pull the code from Git    
+ 
+3. Open Command terminal       
+ 
+4. Go to the client folder inside the project folder (e.g. **\Volunteer-app path\client**)      
+ 
+5. type in command terminal: npm install     
+    >**Note:** Inside client folder there is package-lock.json. Everytime this is modified, it is advised to do step 4      before       running the project.      
+​
+6. type in command terminal: expo start
```
Comment (Boopster) [inline README.md:33]: I think this should be "npm start" which runs the "expo start" script

**Review by Boopster — APPROVED** [review]
I think we should include instructions for running the tests and the node.js project also but can be done in a separate PR

**Review by CatMac228 — APPROVED** [review]

**Comment by gunnx** [comment]
also good to add that you should be installing dependencies with `expo install <dep>` instead of 'npm install <dep>` this ensures that a compatible version is installed.

**Pull request merged**